src/dawn/CodeGen/GridTools/GTCodeGen.cpp

```diff
@@ -629,36 +628,94 @@ std::string GTCodeGen::generateStencilInstantiation(
 
     mplContainerMaxSize_ = std::max(mplContainerMaxSize_, numFields);
 
-    std::vector<std::string> StencilConstructorTemplates = buildFieldTemplateNames(nonTempFields);
+    std::vector<std::string> StencilConstructorTemplates;
+    int numTemporaries = 0;
```

Comment:
use `tempFields` and `nonTempFields` of the fix_kcaches branch

Reply:
ok